packages/aws-cdk/README.md

```diff
@@ -1238,6 +1238,219 @@ that can be set in many different ways (such as `~/.cdk.json`).
 $ # Check the current status of telemetry
 $ cdk cli-telemetry --status
 ```
+### `cdk flags` 
+
+View and modify your feature flag configurations.
+
+Run `cdk flags` to see a report of your feature flag configurations that differ from our recommended states. Unconfigured flags will be labelled with `<unset>` to show that flag currently has no value. The flags are displayed to you in the following order:
+
+1. flags whose states do not match our recommended values
+2. flags that are not configured at all
+
+```shell
+$ cdk flags --unstable=flags
+    Feature Flag                              Recommended                  User
+  * @aws-cdk/...                              true                         false
+  * @aws-cdk/...                              true                         false
+  * @aws-cdk/...                              true                         <unset>
+```
+
+Alternatively, you can also run `cdk flags --all` to see a report of all feature flags in the following order:
+
+1. flags whose states match our recommended values
+2. flags whose states do not match our recommended values
+3. flags that are not configured at all
+
+```shell
+$ cdk flags --unstable=flags --all 
+    Feature Flag                              Recommended                  User
+    @aws-cdk/...                              true                         true
+  * @aws-cdk/...                              true                         false
+  * @aws-cdk/...                              true                         false
+  * @aws-cdk/...                              true                         <unset>
+```
+
+### Modifying your feature flag values
+
+To start, you can run `cdk flags -interactive`  (or `cdk flags --i`) to view the interactive menu options or run `cdk flags --help` to see all options available to you. From here, you can choose to modify your feature flag configurations. After choosing an option or running a command, you see the potential impact on your application and decide if you want to accept or reject the changes. Should you choose to accept, your CDK application resynthesizes with the new feature flag configurations. This means changes are not made until you have confirmed them.
+
+#### Use the interactive menu
```

Comment:
All of this is good, but I think we don't need to show the entire menu. This eats up a lot of space in the guide, and I think we can trust people to navigate their way through a UI.

I think we can do with a single sentence saying that you can also do all these things interactively by passing `-i`.